labelme/config/default_config.yaml

```diff
@@ -13,6 +13,17 @@ labels: null
 file_search: null
 sort_labels: true
 validate_label: null
+label_with_attrs: null # ["your name of label with attributes", "sublabel", "your name of chosenRadioButtonObjAttr",
+                      # "your name of objAttributesTextFields",
+                      # "your name of objAttributesNumericRangeFields"]
+object_attrs_values: null # dict() with keys: 'radio_buttons' or 'text_fields' or 'numeric_range'
+object_attrs_variables: null # dict() {"chosenRadioButtonObjAttr": "your name of attribute",
```

Comment:
label_with_attrs, object_attrs_values, object_attrs_variables should not be null?

Reply:
In the default_config they have to be null. These comments above are directions for the user how to fill out "label_with_attrs, object_attrs_values, object_attrs_variables" in their local config files.